src/stateSpectralLine.ttl

```diff
@@ -31,14 +31,18 @@
 sorelch:hasSubstance rdf:type owl:AnnotationProperty .
```

Comment:
sorelch:hasSubstance is defined as an object property in line 46 of  src/statePhysical.ttl (above)

Reply:
Good catch, OWLAPI didn't find a pun there, maybe neither ontology imports the other.